compiler/rustc_log/Cargo.toml

```diff
@@ -6,8 +6,9 @@ edition = "2021"
 [dependencies]
 atty = "0.2"
 tracing = "0.1.28"
-tracing-subscriber = { version = "0.3.3", default-features = false, features = ["fmt", "env-filter", "smallvec", "parking_lot", "ansi"] }
+tracing-subscriber = { version = "0.3.16", default-features = false, features = ["registry", "fmt", "env-filter", "smallvec", "parking_lot", "ansi"] }
```

Comment:
I didn't check this super carefully so I might not need to bump all the way to .16 but I had to bump the version in order to get support for per layer filtering, this is also what brought in all the new dependencies.

Reply:
looks like we could restrict it to 0.3.15 to prevent it from depending on `nu_ansi_term` but after looking into it I don't see any particular reasons to avoid it. The only real new dep it brought in was overload which seems to be a convenience macro

Reply:
yea seems fine